deploy/cloudformation/manifest-pipeline.yml

```diff
@@ -51,6 +51,14 @@ Parameters:
   RBSCS3ImageBucketArn:
     Type: String
     Description: "The bucket that rarebooks images are in for the search process"
+  GoogleKeyPath:
+    Type: String
+    Description: The ssm path to look for the google team drive credentials and drive-id
+    Default: '/all/marble/google'
+  MuseumKeyPath:
+    Type: String
+    Description: The ssm path to look for the EmbARK museum credentials
+    Default: '/all/marble/museum'
```

Comment:
Are these credentials going to be shared between the test and production stacks of this service? I'd generally recommend having separate credentials for test/prod, but I realize it's not always feasible. If you want to just share them, this should be fine as is.

If you do want to separate, you'll need to also change the deployment pipeline to use different paths for the test stack from the prod stack. You would do this by adding cloudformation parameters to the build command for the test stack https://github.com/ndlib/marble-blueprints/blob/master/deploy/cloudformation/manifest-pipeline-pipeline.yml#L461 and for the prod stack https://github.com/ndlib/marble-blueprints/blob/master/deploy/cloudformation/manifest-pipeline-pipeline.yml#L480.

Reply:
We will be using the same credentials for both test and prod.  There is only one EmbARK server, and the museum images only exist in one place.  Both of these are read-only.